### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by clin2on3mun**
in this milestone:
-created owners table
- created species table
Action done to tables
created owner table with data
owner's table
created species and add data
species table
Remove species column in animal table

removeSpeciescOLUMN

using foreign key to reference table 

query-multiple -tables

### CODE REVIEW

File: README.md
Comment (joskalenda) [inline README.md:1]: - [ ] Please make sure you create a MIT.md file in your app directory and customize its content, then take care of updating your readme mit file link as for now it is redirecting the user to another page

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @clin2on3mun , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ SQL queries
- [ ].. ✅ Readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (clin2on3mun) [reply README.md:1]: HI @joskalenda 
I am not getting changes i will make

**Review by binhussen — APPROVED** [review]
Hi @clin2on3mun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
AgreeSealGIF

## Highlights
- Well Structured Query ✔️
- Descriptive pull request with screenshot ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**